lib/runner/cli/cli.js

```diff
@@ -194,30 +198,38 @@ class CliRunner {
 
     packageInfo = packageInfo || {};
     const usingESM = packageInfo.type === 'module';
-
-    return path.resolve(usingESM ? CliRunner.CONFIG_FILE_CJS : CliRunner.CONFIG_FILE_JS);
+    
+    if (usingESM) {
+      return path.resolve(CliRunner.CONFIG_FILE_CJS);
+    }
+    // TODO: check for both the `nightwatch/tsconfig.json` and `tsconfig.nightwatch.json`
+    if (Utils.fileExistsSync(path.resolve('tsconfig.nightwatch.json'))) {
```

Comment:
should be refactored into a `usingTS` constant, similarly to the `usingESM` one above and I think we should also check if `CliRunner.CONFIG_FILE_TS` exists, because you can use a `.js` or `.json` config file even if you are using  TS.

@garg3133 do we create the `nightwatch.conf.ts` from the init tool?